documentation/feature/javelin.md

```diff
@@ -6,31 +6,30 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### 1.1 Improved locking
+The locking capabilities of the titan and javelin got improved, you can now lock on anything that have a thermal signature including houses.
```

Comment:
"...the Titan and Javelin were improved, you can now lock on anything that has a thermal signature, including houses."
Capitalization, have/has, a comma. :p